native_client/BUILD

```diff
@@ -103,34 +114,26 @@ tf_cc_shared_object(
             ### => Trying to be more fine-grained
             ### Use bin/ops_in_graph.py to list all the ops used by a frozen graph.
             ### CPU only build, libdeepspeech.so file size reduced by ~50%
-            "//tensorflow/core/kernels:dense_update_ops",   # Assign
-            "//tensorflow/core/kernels:constant_op",        # Const
-            "//tensorflow/core/kernels:immutable_constant_op", # ImmutableConst
+            "//tensorflow/core/kernels:dense_update_ops",   # Assign (remove once prod model no longer depends on it)
```

Comment:
Hm, you removed all the `assign` op, are you referring to the prod model we release / host for running tests ?
If so, I do worry that it's a bit burried and we may forgot

Reply:
Yep. If we remove this then prod model tests fail when loading the graph.

Reply:
Maybe we should file an issue to not forget and refer to the issue here?

Reply:
Filed #2179